### ISSUE TITLE
light/dark mode toggle is not inside a landmark (all page content should be contained by landmarks)

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/geniza#1090]

**Comment by rlskoeser** [comment]
confirmed fixed on test site by scanning with axe DevTools
